modules/web/js/ballerina/components/transform-render.js

```diff
@@ -977,7 +959,36 @@ class TransformRender {
             alwaysRespectStubs: true,
         }];
     }
-}
 
 
+    onRemove(id, func, removeFunction, reference) {
+        const self = this;
```

Comment:
Since we are using arrow functions here, we should be able to get rid of 'self' const.